packages/config/config/devices/0x0154/flow_stop_2.json

```diff
@@ -0,0 +1,26 @@
+// Popp & Co Flow Stop
+// Version 2
+{
+	"manufacturer": "Popp & Co",
+	"manufacturerId": "0x0154",
+	"label": "Flow Stop 2",
+	"description": "",
```

Comment:
```suggestion
	"label": "POPE701479",
	"description": "Flow Stop 2",
```

Reply:
It's popp not pope

Reply:
Sure?
http://manuals-backend.z-wave.info/make.php?lang=DE&sku=POPE701479&cert=ZC08-14030002